jcl/src/java.base/share/classes/java/lang/Access.java

```diff
@@ -386,7 +386,7 @@ public String newStringUTF8NoRepl(byte[] bytes, int offset, int length) {
 		/*[IF JAVA_SPEC_VERSION < 17]*/
 		return StringCoding.newStringUTF8NoRepl(bytes, offset, length);
 		/*[ELSE] JAVA_SPEC_VERSION < 17 */
-		/*[IF (JAVA_SPEC_VERSION < 21) | INLINE-TYPES]*/
+		/*[IF JAVA_SPEC_VERSION < 21 ]*/
```

Comment:
Pls remove the extra space, here and in the next change.

Reply:
Removed